.github/workflows/ci.yml

```diff
@@ -21,11 +21,5 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Lint
-        run: npm run lint
-
       - name: Build
```

Comment:
Should we call this workflow `build.yml`?

Reply:
Done!